reframe/frontend/cli.py

```diff
@@ -81,6 +84,12 @@ def main():
         help='Load checks recursively')
 
     # Select options
+    select_options.add_argument(
+        '--modules-mapping', action='store', dest='module_map_file',
```

Comment:
I think `--module-map` is a better name. Also set the `metavar=FILE` so as to have a better description in the help message.